### PR TITLE
fix test for forms/form

### PR DESCRIPTION
**Pull request by bhbs**
```sh
cd forms/form
cargo test
```

### CODE REVIEW

File: forms/form/src/main.rs [hunk 1]
```diff
@@ -186,7 +186,7 @@ mod tests {
             name: "John".to_string(),
         });
         let result = handle_post_3(req.clone(), params).await;
-        let resp = result.respond_to(&req);
+        let resp = result.customize().respond_to(&req);
```
Comment (robjtede) [inline forms/form/src/main.rs:189]: why is this necessary? no customization is being done.

Comment (bhbs) [reply forms/form/src/main.rs:189]: I'm sorry, this might be a beginner question, but I am getting this error.
How can it be suppressed?

Comment (bhbs) [reply forms/form/src/main.rs]: Thanks for the review. Then I will leave it untouched in this pull request.

**Review by robjtede — APPROVED** [review]
